packages/apps/human-app/frontend/src/shared/components/data-entry/input-masks/input-masks.ts

```diff
@@ -0,0 +1,9 @@
+import { HumanCurrencyInputMask } from '@/shared/components/data-entry/input-masks/human-currency-input-mask';
```

Comment:
Certainly! I look through the files though and it looks like a bigger change than re-export, as the change You requested impacts a lot of input fields, which need to be tested. WDYT to include it in next PRs? This PR is pretty big itself and hiding this change could get lost in the history :sweat_smile:

Reply:
@mpblocky indeed, since we rely on generic `InputMasks[string]` - it will impact many places with big refactor and doesn't make a lot of sense atm.

Let's then keep `InputMasks` object then, but just rename this file to `index.ts` because `data-entry/input-masks/input-masks.ts` looks weird

Reply:
Fixed!